### ISSUE TITLE
Zsh: Bash prompt displayed in status after command execution

### ISSUE THREAD

**Issue by arialdomartini** [hydrargyrum/asciinario#2]
Right after a command is executed, the top status line is replaced with the Bash prompt.
For example, running

    ./asciinario.py my.asciinario my.cast

on the following Scenario

```
set type_wait = 0.1
status show top
-> write "echo hello" and hit return
wait 10
```

generates a Asciinema cast including the following extra line

```
[5.938785, "o", "\u001b[H\u001b[7marialdo@mbuto:~/tmp/asciinario                                                             \r\n\n\n\u001b[27m"]
```

This line replaces the string

    write "echo hello" and hit return

with my standard Bash status line

    arialdo@mbuto:~/tmp/asciinario

See the resulting cast here:

https://asciinema.org/a/Fo6fjnxmlDiOiJtTDvCtLe1ER

This only happens when Asciinario is run from Zsh. If from Zsh I first run a `bash`, then all works as expected. So, that's no blocking issue: one should just make sure to run Asciinario from Bash, not from Zsh.

**Comment by hydrargyrum** [comment]
It's a feature from your shell configuration that attempts to set the window title with something (the prompt in your case) by using an escape sequence. It's normally caught by your terminal emulator, but screen interprets it and sets its status bar with it.

The sequence is `\033]1;TITLE\007` or `\033]2;TITLE\007` (at least that's what `xtitle` emits)

I will try to make screen ignore those sequences.

**Comment by hydrargyrum** [comment]
@arialdomartini can you try again? I merged 89fe6f002

**Comment by arialdomartini** [comment]
Yes, it works! Thank you!
